packages/core/core-flows/src/order/workflows/return/create-return-shipping-method.ts

```diff
@@ -142,6 +145,7 @@ export const createReturnShippingMethodWorkflow = createWorkflow(
         "claim_id",
         "exchange_id",
         "canceled_at",
+        "items.*",
```

Comment:
nit: Can we be more specific about the fields we choose? No need to fetch everything if we can avoid it.

Reply:
Are these items even used?

Reply:
good catch, initially it was used but then forgot to remove